update_vendored_collections/update_vendored_collections.py

```diff
@@ -87,14 +87,26 @@ def spec_replace_sources(content, collection_tarballs):
     return content
 
 
+def spec_replace_bundled_provides(content, collection_tarballs):
+    print("Replacing bundled provides in the spec file")
+    for collection, tarball in collection_tarballs.items():
+        version = re.sub(".tar.gz", "", re.sub(".*-", "", tarball))
```

Comment:
```suggestion
        version = re.sub("[.]tar.gz", "", re.sub(".*-", "", tarball))
```
Use `[.]` to ensure matching a literal `.` (you could also use `\`)

Reply:
Thank you, done. Flake was complaining about such escaping, I needed to add `r` to the string like `r"\.tar\.gz"`.